### ISSUE TITLE
Missing writing permissions in version 2.0.1

### ISSUE THREAD

**Issue by cru8602** [ioBroker/ioBroker.node-red#121]
Since version 2.0.1 it's possible to reactivate the palette manager in oder to install nodes via node red. But when I try to install a new node or update an already installed one I get a lot of errors for missing writing permissions. Can this pls be fixed (or can somebody tell me which permissions are needed and how to set them?(sry if this is a noob question))

**Comment by Apollon77** [comment]
Hm ... WHat about concrete error logs? What about more details to who the owner of the affected files is currently?

**Comment by cru8602** [comment]
I was able to solve the problem with chmod.
The folders in question were subfolders of the iobroker-data/node-red/node_modules folder which I manually copied because the node installation via iobroker didn't work. I guess I didn't used the iobroker user for that. 
The node I tried to install was also a node that I had copied before but went missing due to the update. The folder of that node was still in place an the iobroker user wasn't allowed to write in it.
